crates/swc_common/src/comments.rs

```diff
@@ -548,6 +548,16 @@ pub struct Comment {
     pub text: String,
 }
 
+impl Default for Comment {
```

Comment:
default was added to satisfy mem::take in `VersionedSerializable::take`. if there's other way default is not required mandatoriliy.

I'm bit unsure if I understood the suggestion correctly, mind elaborate bit more? The way I understood is let each struct implement `Take::dummy()` instead of default, and VersionedSerializable should utilize it for the `take`. Not sure where `into_inner` plays in here.

Reply:
`into_inner` take ownership of `VersionedSerializable` so it does not require `take(&mut self.0.1)`.

Reply:
The `Default` constraint comes from it.